tests/test_files/starter_preset_patcher.json

```diff
@@ -272,12 +273,12 @@
         },
         {
             "pickup_type": "actor",
-            "caption": "Morph Ball acquired.",
+            "caption": "Sent Speed Booster to meep.",
```

Comment:
Should have been a progressive jump 😥
Was it intentional to change the **starter_preset** file with different start point, multiworld stuff and the debug thing `reveal_map_on_start` ?

Reply:
Oh whoops no that was supposed to be reverted